inline/src/lexer/token.rs

```diff
@@ -788,33 +749,47 @@ impl Span {
     ///
     /// Tuple containing the resulting span and the removed span.
     ///
-    /// [`Span`]: self::Span
+    /// [`Span`]: unimarkup_commons::scanner::span::Span
+    fn remove(self, other: Span) -> (Span, Span);
+    fn swapped(&self, other: &Span) -> (Span, Span)
```

Comment:
```suggestion
    fn swap(&self, other: &Span) -> (Span, Span)
```

rename as in `commons/src/scanner/span`.

Reply:
Oh if there's `Span` anywhere else left it must be deleted completely! Nice catch!

Reply:
For now I will move the function to `SpanExt` trait, because we don't use that anywhere else. If necessary, we can always move it back.